### ISSUE TITLE
groff is uninstallable on linux

### ISSUE THREAD

**Issue by bbatha** [Homebrew/linuxbrew-core#21535]
This commit https://github.com/Homebrew/linuxbrew-core/commit/4ff488ffc18315c31824df560a19e627cc656a57#diff-acc814322b026f2731b20e15ad2c62d209a6a6ae25610649fc907da432be0fb6 added the `uses_from_macos "iconv"` line which has made groff incompatible with linux as iconv is part of libc on linux. 

```
hostname:~/dev$ brew install groff
libiconv: is not needed on Linux, where iconv.h is provided by glibc
Error: An unsatisfied requirement failed this build.
```

**Comment by ChrisWiegman** [comment]
This looks to be another related to #21533

**Comment by iMichka** [comment]
Working on a fix here: #21539

**Comment by dawidd6** [comment]
Fixed by https://github.com/Homebrew/linuxbrew-core/pull/21539
